### ISSUE TITLE
`Offset`, `limit` and `any` not working in parser

### ISSUE THREAD

**Issue by joshstevens19** [oguimbal/pgsql-ast-parser#65]
A few script which i thought would pass in this lib seems to fail.. 

Broken scripts:

doesn't work: 

```
SELECT *
FROM profile_view
OFFSET 0 LIMIT 10;
```

^^ script works in postgres if you run it

doesn't work:

```
SELECT *
FROM profile_view
WHERE handler = ANY (array['testme'])
```

^^ script works in postgres if you run it

this is using:

`import { parse } from 'pgsql-ast-parser';` method

**Comment by oguimbal** [comment]
Hi, thanks for the feedback.

I didnt realized that `OFFSET x LIMIT y` could be ordered that way 😄   (`LIMIT y OFFSET x` works)

Will fix !

**Comment by joshstevens19** [comment]
Awesome can reverse to fix thanks

How about the any clause?

**Comment by oguimbal** [comment]
Offset & limit fixed `pgsql-ast-parser@9.3.0`.

I did not understand the "any" part... the statement provided parses correctly  🤷‍♂️
